invenio_communities/views/api.py

```diff
@@ -101,8 +102,37 @@ def get(self, query, sort, page, size):
             :resheader Content-Type: application/json
             :statuscode 200: no error
         """
+        urlkwargs = {
+            'q': query,
+            'sort': sort,
+            'size': size,
+        }
+
+        query = Community.filter_communities(query, sort)
+
+        endpoint = '.communities_list'
+        links = {
+            'self': url_for(endpoint, page=page, _external=True, **urlkwargs),
+        }
+
+        if page > 1:
```

Comment:
if you store result of `query.paginate(page, size)` to `pagination`, then you can use `pagination.has_prev` and `pagination.has_next` properties.